Jenkinsfile_CNP

```diff
@@ -30,6 +33,7 @@ GradleBuilder builder = new GradleBuilder(this, product)
 withPipeline(type, product, component) {
   def githubApi = new GithubAPI(this)
   env.IDAM_S2S_AUTH_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+  env.IDAM_BEARER_AUTH_URL = "https://idam-api.aat.platform.hmcts.net"
```

Comment:
Can we name it `IDAM_API_URL` please, it should be existing env var with in this repo.